.buildkite/scripts/steps/code_coverage/reporting/ingestData.sh

```diff
@@ -49,4 +49,11 @@ ingestModular() {
 
   echo "---  Ingesting Code Coverage - Complete"
   echo ""
+
+  currentBuildNumber="$BUILDKITE_BUILD_NUMBER"
+  url="https://kibana-stats.elastic.dev/app/discover#/?_g=(filters:!(),query:(language:kuery,query:''),refreshInterval:(pause:!t,value:0),time:(from:now-7d,to:now))&_a=(columns:!(),filters:!(),hideChart:!f,index:'64419790-4218-11ea-b2d8-81bcbf78dfcb',interval:auto,query:(language:kuery,query:'BUILD_ID%20:%20${currentBuildNumber}'),sort:!(!('@timestamp',desc)))"
+
+  echo "--- Browse the following url to visually verify in Kibana (Disover):"
+  printf "%s \n" "$url"
```

Comment:
If there's any interest in putting this in a markdown box at the top of the build you can do something like https://github.com/elastic/kibana/blob/f13de61291e13b8625e71fb694c125752d5d4a64/.buildkite/scripts/steps/artifacts/publish.sh#L78-L82

output: https://buildkite.com/elastic/kibana-artifacts-snapshot/builds/285#annotation-artifacts-summary